telemetry/telemetry_record.py

```diff
@@ -73,15 +100,48 @@ def move_wpr_files(name_schema):
     """
     remote_data_path = os.path.join(CHROMIUM_SRC, 'tools/perf/page_sets/data/',
             name_schema)
-    local_path = os.path.join(os.getcwd(), 'data/')
+    local_path = os.path.join(os.getcwd(), 'data/wpr_source')
     p = Popen('mv {0} {1}'.format(remote_data_path, local_path), shell=True)
 
+def make_modified_json(num_urls):
+    """Creates *.json files to point to the right archive files
+
+    Located in ~/src/tools/perf/page_sets/data
+    :param num_urls: int for the number of target urls
+    TODO: add sha1?
+    """
+    json_path = '/home/jamshed/src/tools/perf/page_sets/data/'
```

Comment:
Cool thanks. Just a high level comment (what you did is totally fine for
now): relative might be better than absolute, since we don't have to
hardcode anything -- as long as you invoke the script from the top-level of
wherever you put the repository. WDYT?

On Mon, Apr 13, 2015 at 9:55 PM, Jamshed Vesuna <notifications@github.com>
wrote:

> In telemetry/telemetry_record.py
> <https://github.com/colin-scott/page_load_time/pull/2#discussion_r28300626>
> :
>
> >      p = Popen('mv {0} {1}'.format(remote_data_path, local_path), shell=True)
> >
> > +def make_modified_json(num_urls):
> > +    """Creates *.json files to point to the right archive files
> > +
> > +    Located in ~/src/tools/perf/page_sets/data
> > +    :param num_urls: int for the number of target urls
> > +    TODO: add sha1?
> > +    """
> > +    json_path = '/home/jamshed/src/tools/perf/page_sets/data/'
>
> Sorry, this is me being lazy. I've updated all relative paths to absolute,
> and all absolute paths to reference PLT_SRC as
> /home/jamshed/page_load_time/ and CHROMIUM_SRC from now on.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/colin-scott/page_load_time/pull/2/files#r28300626>.
>